src/app/features/yang-search/yang-search.component.ts

```diff
@@ -244,6 +249,12 @@ export class YangSearchComponent implements OnInit, OnDestroy, AfterViewInit {
       takeUntil(this.componentDestroyed)
     ).subscribe(
       results => {
+        if (results['max-hits']) {
+          this.warnings.push('Maximum number of results reached. Not all results will be shown.')
+        }
+        if (results['timeout']) {
+          this.warnings.push('Timeout while searching. Please try searching for something more specific.')
```

Comment:
These 2 messages can be displayed directly in ngb-alert component -> no need to open modal to display these messages.